sycl/include/CL/sycl/handler.hpp

```diff
@@ -668,7 +668,7 @@ class __SYCL_EXPORT handler {
       KernelType MKernelFunc;
       NormalizedKernelType(const KernelType &KernelFunc)
           : MKernelFunc(KernelFunc) {}
-      void operator()(const nd_item<Dims> &Arg) {
+      void operator()([[maybe_unused]] const nd_item<Dims> &Arg) {
         detail::runKernelWithoutArg(MKernelFunc);
```

Comment:
The conventional way to fix this warning is to cast to `void`. I suppose maybe_unused attribute might still be unsupported by some compilers.
```suggestion
      void operator()(const nd_item<Dims> &Arg) {
        (void)Arg;
        detail::runKernelWithoutArg(MKernelFunc);
```

Reply:
Can do.